Tests/test_phenotype.py

```diff
@@ -142,11 +144,16 @@ def test_PlateRecord(self):
         p2 = p.subtract_control()
         self.assertEqual(p2.id, p.id)
         self.assertEqual(p2["A02"], p["A02"] - p["A01"])
-        self.assertEqual(repr(p), "PlateRecord('WellRecord['A01'], WellRecord"
-                         "['A02'], WellRecord['A03'], ..., WellRecord['B12']')")
-        self.assertEqual(str(p), "Plate ID: PM01\nWell: 24\nRows: 2\nColumns: "
-                         "12\nPlateRecord('WellRecord['A01'], WellRecord['A02'], WellRecord"
-                         "['A03'], ..., WellRecord['B12']')")
+        self.assertEqual(
+            repr(p),
+            "PlateRecord('WellRecord['A01'], WellRecord"
+            "['A02'], WellRecord['A03'], ..., WellRecord['B12']')",
+        )
+        self.assertEqual(
+            str(p),
+            "Plate ID: PM01\nWell: 24\nRows: 2\nColumns: 12\nPlateRecord('WellRecord"
+            "['A01'], WellRecord['A02'], WellRecord['A03'], ..., WellRecord['B12']')",
+        )
```

Comment:
Can you break the string literal at one of the spaces instead? It would be a small improvement to both assert-equals calls.

Reply:
Done, Thank you.